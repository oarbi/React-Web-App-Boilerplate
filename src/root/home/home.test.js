import React from 'react';
import { shallow } from 'enzyme';
import Home from './home.component';

describe('Home Component', () => {
  it('should render correctly', () => {
    const user = {
      name: {
        first: 'Brad',
        last: 'Gibson',
      },
      picture: {
        large: 'https://randomuser.me/api/portraits/men/75.jpg',
      },
      email: 'brad.gibson@example.com',
      phone: '011-962-7516',
    };
    const component = shallow(<Home user={user} />);

    expect(component).toMatchSnapshot();
  });
});
