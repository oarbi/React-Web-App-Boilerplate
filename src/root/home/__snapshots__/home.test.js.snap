// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should render correctly 1`] = `
<div
  className="home"
>
  <div
    className="home__language-picker"
  >
    <span
      className="home__language-picker__language-btn"
      onClick={[Function]}
    >
      EN
    </span>
    <span
      className="home__language-picker__language-btn"
      onClick={[Function]}
    >
      FR
    </span>
  </div>
  <div
    className="home__profile"
  >
    <img
      alt="avatar"
      className="home__profile__avatar"
      src="https://randomuser.me/api/portraits/men/75.jpg"
    />
    <h1>
      greeting
    </h1>
    <p>
      <span
        className="home__profile__label"
      >
        email
        :
      </span>
       
      brad.gibson@example.com
    </p>
    <p>
      <span
        className="home__profile__label"
      >
        phone
        :
      </span>
       
      011-962-7516
    </p>
  </div>
</div>
`;
